### PR TITLE
Run tests on xcore-ai

### PR DESCRIPTION
**Pull request by lucianomartin**
- use template from lib_logging: xmos/lib_logging#42
- use new functions from xmos jenkins shared lib v0.16.0

This PR includes the changes in https://github.com/xmos/lib_dsp/pull/182

### CODE REVIEW

File: tests/test_fft_forward_real/LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Software Release License Agreement
 
-Copyright (c) 2016-2017, XMOS, All rights reserved.
+Copyright (c) 2016-2020, XMOS, All rights reserved.
```
Comment (mbanth) [inline tests/test_fft_forward_real/LICENSE.txt:3]: 2020?

**Review by mbanth — CHANGES_REQUESTED** [review]
One small problem.

File: tests/test_fft_forward_real/LICENSE.txt (hunk 1 above)
Comment (lucianomartin) [reply tests/test_fft_forward_real/LICENSE.txt]: The original commit changing the folder with the license was done in 2020, this is the reason why 2020 was there and the license check didn't fail.

I have updated the date to 2021.

**Review by mbanth — DISMISSED** [review]
Looks good.

**Review by mbanth — APPROVED** [review]
Looks good.

**Pull request merged**